localization/english/dynasties/wc_dynasty_names_l_english.yml

```diff
@@ -6918,4 +6916,4 @@
  dynn_Kosumothid:0 "Kosumothid"
  dynn_Yenajzid:0 "Yenajzid"
  dynn_Erudaxid:0 "Erudaxid"
- dynn_Zakajzid:0 "Zakajzid"
\ No newline at end of file
+ dynn_Zakajzid:0 "Zakajzid"l_english:
```

Comment:
is this intended?

Reply:
Fixed in b279cfe17d70a9ca4d9bd3382b71f371cdac6613.